### PR TITLE
Add build/install steps for Debian bookworm

### PR DESCRIPTION
**Pull request by owenh000**
Add easy and concise instructions for building and installing on Debian bookworm.

### CODE REVIEW

**Comment by owenh000** [comment]
While there is a lot of helpful information in the main README, I find that only a few simple steps are actually required to get ScanTailor working on Debian bookworm. Having those steps listed together makes the process must faster, at least for me.

**Comment by mara004** [comment]
Shorthand instructions for a specific distro may be helpful for some users, yet I don't think they belong in this project.
All this info is already available in the Linux and Packaging sections in a fairly straightforward manner. I don't like the redundancy, this only poses a maintenance problem (risk of desync with main instructions).

TLDR your code may be nice as a gist but IMHO it should not be merged here.

**Comment by owenh000** [comment]
@mara004, feel free to copy the contents of this pull request to a gist or whatever if you wish to. I am happy to contribute this small amount of documentation ([CC0](https://creativecommons.org/share-your-work/public-domain/cc0/)) with the hope it might be useful to someone. If it is merged into the community project, it may perhaps in the future receive maintenance updates by other volunteers, rather than being lost due to fragmentation of code/documentation (as is already so common due to divergent forks on GitHub).

Another option is cleaning up the main instructions and improving the distribution-specific content so a regular user can have a working ScanTailor in just a few minutes plus build time without having to parse lots of content that isn't needed for that. Please don't ask me to do it because I am not able to at this time.

Meanwhile, I'll leave this for @lightsilverberryfox or @vigri to accept or reject.

**Comment by mara004** [comment]
> Another option is cleaning up the main instructions and improving the distribution-specific content so a regular user can have a working ScanTailor in just a few minutes plus build time without having to parse lots of content that isn't needed for that.

Again, IMO the main instructions are fine. You don't have to "parse lots of content", the Table of contents is really pretty straightforward, and with a quick skim you're at the Linux section.

I agree the main Readme could get some more copy-paste distro package install lines, but in the meantime it shouldn't be too difficult to search for the names listed in the Dependencies section and manually find the corresponding distro packages.

But yes, this is not for me to decide. Just giving my opinion.
